### DISCUSSION TITLE
Welcome to insar-explorer Discussions!

### DISCUSSION THREAD

**Discussion by mahmud1** [luhipi/insar-explorer#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 2diagon** [comment]
Is there a way for the graph to display a legend with the IDs of the time series being plotted?
It would be very useful to visually confirm that the ID corresponds to the intended element, this would greatly improve result interpretation when working with multiple elements.
Thank you very much for your work on this plugin.

**Reply by mahmud1** [reply]
Great suggestion. I'll look into adding this in a future release. Thanks for the kind words about the plugin and appreciate the feedback!
